.cruft.json

```
@@ -7,14 +7,14 @@
       "name": "TopoLVM",
       "slug": "topolvm",
       "parameter_key": "topolvm",
-      "test_cases": "defaults",
+      "test_cases": "defaults legacy",
       "add_lib": "n",
       "add_pp": "n",
       "add_golden": "y",
       "add_matrix": "y",
       "add_go_unit": "n",
       "copyright_holder": "VSHN AG <info@vshn.ch>",
-      "copyright_year": "2022",
+      "copyright_year": "2023",
```

You'll probably want to adjust this to 2021 so that the Cruft config matches the `LICENSE` file

```suggestion
      "copyright_year": "2021",
```